packages/data/src/redux-store/metadata/reducer.ts

```diff
@@ -142,8 +155,9 @@ const isResolved = ( state: Record< string, State > = {}, action: Action ) => {
 		case 'FAIL_RESOLUTIONS':
 		case 'INVALIDATE_RESOLUTION':
 			return subKeysIsResolved( state, action );
+		default:
+			return state;
 	}
-	return state;
```

Comment:
TS removed that `return state;` at the end, considering the switch cases to be exhaustive. Thus I added it as a default case